ogc_ws/src/identifiers/scripts/identifiers.py

```diff
@@ -371,7 +371,7 @@ def edit_device(self, device):
 		selected_device["imei"] = device["imei"] if device["imei"] != ""else selected_device["imei"]
 		selected_device["rb_serial"] = device["rb_serial"] if device["rb_serial"] != "" else selected_device["rb_serial"]
 		selected_device["telegram_id"] = device["telegram_id"] if device["telegram_id"] != "" else selected_device["telegram_id"]
-		selected_device["syncthing_id"] = device["syncthing_id"] if device["syncthing_id"] != "" else selected_device["syncthing_id"]
+		# selected_device["syncthing_id"] = device["syncthing_id"] if device["syncthing_id"] != "" else selected_device["syncthing_id"]
```

Comment:
@rumesh986 can you check on this? Latest updates from the syncthing issue is that this was modified to fix issues with the admin server

Reply:
fixed along with the merge conflict